case_study/memory_leaks/images_1/lib/main.dart

```diff
@@ -36,6 +36,20 @@ class MyHomePage extends StatefulWidget {
   MyHomePageState createState() => MyHomePageState();
 }
 
+int globalObjectId = 0;
+
+class Tracker {
```

Comment:
nit: can we use a different name as this sounds like the name of the feature it is testing which makes the UI confusing. Maybe just something simple like
ObjectWithUniqueId.
I think my confusion in previous comments was from seeing the word Tracker in the table.

Reply:
Done.